scripts/dotnet-cli-build/Utils/Utils.cs

```diff
@@ -96,7 +96,7 @@ public static void DeleteDirectory(string path)
                     File.SetAttributes(file, FileAttributes.Normal);
                     File.Delete(file);
                 }
-
+                System.Threading.Thread.Sleep(1); 
```

Comment:
Solved the issue when all files inside directory were deleted but directory deletion failed with "non empty" error.